python/morpheus_llm/morpheus_llm/requirements_morpheus_llm.txt

```diff
@@ -0,0 +1,11 @@
+# This file is generated by `rapids-dependency-file-generator`.
+# To make changes, edit ../../../dependencies.yaml and run `rapids-dependency-file-generator`.
+--extra-index-url https://download.pytorch.org/whl/cu124
+faiss-cpu
```

Comment:
Will these be checked to be up to date by CI?

Reply:
Yes. ci/check_style.sh does a pre-commit run and reports changed files. And pre-commit runs the dependency-file-generator.